packages/editable-html/src/plugins/list/__test__/index.test.js

```diff
@@ -0,0 +1,152 @@
+import React from "react";
+
+import {configure} from "enzyme";
+import Adapter from "enzyme-adapter-react-15";
+import renderer from 'react-test-renderer';
+import List from './../../list/index';
+import Util from '../../utils';
+
+configure({adapter : new Adapter() });
+
+describe("ListPlugin", () => {
+  let list, ListPlugin, toolbar;
+  beforeEach(() => {
+    list = require('../index');
+    ListPlugin = list.default;
+    toolbar = list.toolbar
+  });
+
+  describe("deserialize", () => {
+    it("should deserialize list item", () => {
```

Comment:
There's a fair bit of boilerplate in the following tests, could we add a shared function?

```javascript
const assertDeserialize = (tagName, expectedType) => {
  it(`deserializes: ${tagName} to ${expectedType}`, () = > {
    const out = serialization.deserialize({tagName}, jest.fn());
    expect(out).toEqual({object: 'block', type:expectedType, nodes: undefined});
  });
}

assertDeserialization('li', 'list_item');
assertDeserialization('ul', 'ul_list');
assertDeserialization('ol', 'ol_list');

```

Reply:
Done